i18n-msg-behavior.html

```diff
@@ -97,6 +101,13 @@
           value: {}
         },
 
+        /**
+         * i18n object, by default it contains all translations.
+         */
+        i18n: {
+          type: Object,
+          value: {}
```

Comment:
This object will be shared across element instances that use the behavior. Is that what you want?